docs/compose/PageIndicator/README.md

```diff
@@ -0,0 +1,28 @@
+# Page Indicator
+
+[![Maven Central](https://img.shields.io/maven-central/v/net.skyscanner.backpack/backpack-compose)](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose)
+[![Class reference](https://img.shields.io/badge/Class%20reference-Android-blue)](https://backpack.github.io/android/backpack-compose/net.skyscanner.backpack.compose.pageindicator)
+[![Source code](https://img.shields.io/badge/Source%20code-GitHub-lightgrey)](https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/pageindicator)
+
+## Default
+
+| <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/PageIndicator/screenshots/default.png" alt="PageIndicator component" width="375" /> | <img src="https://raw.githubusercontent.com/Skyscanner/backpack-android/main/docs/compose/PageIndicator/screenshots/default_dm.png" alt="PageIndicator component - dark mode" width="375" /> |
```

Comment:
tiny thing: the day/night headings of the table are missing (which i realise now is an issue with the component generation script. i'll fix that

Reply:
thanks for spotting it! fixed